WordPressComKitTests/SiteServiceTests.swift

```diff
@@ -1,110 +1,55 @@
 import XCTest
-import OHHTTPStubs
-import WordPressComKit
+@testable import WordPressComKit
 
 class SiteServiceTests: XCTestCase {
-    var subject: SiteService!
+    let subject = SiteService()
```

Comment:
As mentioned in the [previous comment](https://github.com/Automattic/WordPressComKit/pull/1/files/f979ee1d14aa027791edaabc726963e92c6cdc31#r57445084), with this being an immutable struct, you can instantiate it all you want, but you'll effectively get an equivalent _value_.